pkg/ccl/changefeedccl/changefeed_stmt.go

```diff
@@ -318,35 +319,80 @@ func changefeedPlanHook(
 			}
 
 			recordPTSMetricsTime := sliMetrics.Timers.PTSCreate.Start()
-
+			// ptr is the feed-level protected timestamp record which exists when per-table protected timestamps are disabled.
```

Comment:
nit: line length https://cockroachlabs.atlassian.net/wiki/spaces/CRDB/pages/181371303/Go+Golang+coding+guidelines#Line-Length

ditto for the comment below